docs/configs/docsdev.js

```diff
@@ -433,7 +433,7 @@ export default {
                                 link: '/zh-cn/docs/dev/user_doc/guide/task/dependent.html',
                             },
                             {
-                                title: 'Stored Procedure',
+                                title: 'PROCEDURE',
```

Comment:
```suggestion
                                title: 'Procedure,
```

Reply:
If you change the sidebar title, you should also change the title in content `Stored Prcedure`